language/diem-framework/modules/0L/TowerState.move

```diff
@@ -19,9 +19,10 @@ module TowerState {
     use 0x1::ValidatorConfig;
     use 0x1::VDF;
     use 0x1::Vector;
+    // use 0x1::Decimal;
 
-    const EPOCHS_UNTIL_ACCOUNT_CREATION: u64 = 14;
-
+    const EPOCHS_UNTIL_ACCOUNT_CREATION: u64 = 13;
```

Comment:
I think this changes is related to use > operator instead of >=

Reply:
If we maintain this const name it would be better to have the value 14 so when someone creates a query for it the display will make sense without the need to add 1